src/pages/clouddocs/annotator_attribute_detection.md

```diff
@@ -5,19 +5,13 @@ categories: Annotators
 slug: annotator_attribute_detection
 toc: true
 ---
-<!-- ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2011, 2023                -->
```

Comment:
This one had an original date of 2015. Not sure if we want to retain that or not?